packages/theme-chalk/src/table.scss

```diff
@@ -27,6 +27,7 @@
   }
 
   @include e(empty-text) {
+    line-height: 60px;
```

Comment:
Comment a line and explain why to make this pr.

Reply:
The empty text of the `min-height:60px` is not taking effect in IE10+, so the PR as workaround to make line-hight same as `min-height` so that it works well under IE10+.

But there is potential issue that is if the height of empty string over 60px, it will cause the issue on other browsers(even though it not likely happen in practice).

As I commented in the PR, it's not bug of Element itself, instead of IE10-IE11 have bug with flexbox with min-height property.

Reply:
用英语没有正确出表达我的想法。😰
我的意思这个修改是只为修复 IE 添加的。那么可以在上面加上一行注释，像下面这样。
```
// min-height doesn't work in IE https://github.com/philipwalton/flexbugs#3-min-height-on-a-flex-container-wont-apply-to-its-flex-items
line-height: 60px;
```

Reply:
谢谢，我修改一下，添加相应的注释。

Reply:
@imzjy 这个的 PR 多了几个 commit，先 rebase，再强推。提交成功后，就可以合并了。😀

Reply:
我已经rebase过了:)

Reply:
@imzjy 看来是不行了。试试 reset 吧，再提交。

Reply:
@ziyoung 
是哦，虽然更新的两个commit，但是sha变了。
我又reset了一个从新rebase了，你再帮我看一下。